app/kubemci/pkg/gcp/backendservice/backendservicesyncer.go

```diff
@@ -174,6 +211,12 @@ func (b *BackendServiceSyncer) desiredBackendService(lbName string, port ingress
 		Port:         port.Port,
 		PortName:     portName,
 		Backends:     desiredBackends(igLinks),
+		// We have to fill in these fields so we can properly compare to what's returned to us
+		ConnectionDraining:  &compute.ConnectionDraining{},
```

Comment:
We should get these values from existing if it exists and skip them otherwise, rather than hardcoding them here.
Please add a TODO to fix this